bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/Constants.kt

```diff
@@ -3,4 +3,7 @@ package io.bucketeer.sdk.android.internal
 internal object Constants {
   const val PREFERENCES_NAME = "bucketeer"
   const val PREFERENCE_KEY_USER_EVALUATION_ID = "user_evaluations_id"
+  const val PREFERENCE_KEY_FEATURE_TAG = "bucketeer_feature_tag"
+  const val PREFERENCE_KEY_EVALUATED_AT = "bucketeer_evaluated_at"
+  const val PREFERENCE_KEY_USER_ATTRIBUTES_UPDATED = "bucketeer_user_attributes_updated"
```

Comment:
Do we need `bucketeer_` prefix?
We created a SharedPreferences with `bucketeer` name, which is just for bucketeer SDK.

Reply:
I see, let me remove the prefix

Reply:
removed https://github.com/bucketeer-io/android-client-sdk/pull/88/commits/9fb9bae0c303d06e14031d339c7bd3e321d3ea17